pkg/polaris/cloudcluster/cloudcluster.go

```diff
@@ -230,19 +372,19 @@ func (a API) CreateCloudCluster(ctx context.Context, input cloudcluster.CreateAw
 		case gqlevent.ActivityStatusQueued:
 		case gqlevent.ActivityStatusRunning:
 		case gqlevent.ActivityStatusTaskSuccess:
-			a.log.Printf(log.Info, "AWS cloud cluster create in progress: %s\n", activitySeries.Activities.Nodes[0].Message)
+			a.log.Printf(log.Info, "cloud cluster create in progress: %s\n", activitySeries.Activities.Nodes[0].Message)
 			time.Sleep(60 * time.Second)
 			continue
 		case gqlevent.ActivityStatusSuccess:
 			return CloudCluster{
 				ID:             uuid.MustParse(activitySeries.ClusterUUID),
```

Comment:
We should parse this with the `uuid.Parse` function and return an error if it fails. The RSC APIs sometimes return `""` for UUIDs when the call fails, if that happens this code would panic.